examples/experimental/onnx_ptq_classification.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import torchvision
+from torchvision import transforms
+
+import onnx
+
+from nncf.experimental.post_training.api.dataloader import DataLoader
+from nncf.experimental.post_training.compressed_model import CompressedModel
+from nncf.experimental.post_training.compression_builder import CompressionBuilder
+from nncf.experimental.post_training.quantization.algorithm import PostTrainingQuantization
+from nncf.experimental.post_training.utils import export
+
+from nncf.experimental.post_training.quantization.algorithm import PostTrainingQuantizationParameters
+from nncf.experimental.post_training.initialization.algorithm import InitializationAlgorithms
+from nncf.experimental.post_training.initialization.quantizer_range_finder import QuantizerRangeFinderParameters
+
+
+def create_train_dataset(dataset_dir):
+    image_size = [224, 224]
```

Comment:
I think this should be revised using two possible options:
1. It can be moved to the constructor of DataLoader
2. It can be kept as is and some helper, like `from_torch_dataset`, is introduced instead of `MyDataLoader`